### DISCUSSION TITLE
This page is taking too long to load.

### DISCUSSION THREAD

**Discussion by jimgwhit** [community/community#37657] [Other Feature Feedback, Questions, & Ideas]
When using the link for my Github gist:

https://gist.github.com/jimgwhit

Instead of it coming up I keep getting another page that says:

> This page is taking too long to load.

**Comment by david-engelmann** [comment]
Restart connection, it appears to be functional

**Comment by jfullstackdev** [comment]
did not have any issue directly opening it from the link you provided

maybe it's on your end, like low memory or slow Internet connection, local computer settings

**Comment by LUIS9799** [comment]
No issues found
